worker.py

```diff
@@ -1535,5 +1814,6 @@ def __graceful_stop(self, stop_trigger: StopSignal):
         # If a restart has been requested...
         # Do nothing.
         # Close the database session
+        self.session.close();
         # End the process
-        sys.exit(0)
+        sys.exit(0)
```

Comment:
Files should end with a newline.

```suggestion
        sys.exit(0)

```